pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "workflows"
+version = "0.1.0"
+description = ""
+authors = ["cern-sis <cern-sis-ts-admin@cern.ch>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+apache-airflow = {version = "2.6.0", extras = ["celery", "postgres", "redis"]}
```

Comment:
isn't it the `2.6.1` the latest?

Reply:
it is, but when we started this PR it was 2.6.0. Let's change to 2.6.1